1209/4646/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: SmartPID SPC1000
```

Comment:
understood
would prefer "SmartPID SPC1000" so better to remove SmartPID from the tile